automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -263,9 +263,10 @@ def product_upgrade(
         sys.exit(1)
     if not os.environ.get('SATELLITE_HOSTNAME'):
         if not sat_image and not os.environ.get('RHEV_SATELLITE'):
-            print('Please provide either Satellite RHEVM template name or '
-                  'Satellite HostName to perform upgrade!')
-            sys.exit(1)
+            sat_image = os.environ.get('SAT_IMAGE')
+            sat_host = os.environ.get('SAT_HOST')
```

Comment:
these variables `SAT_IMAGE` and `SAT_HOST` are defined in jenkins under `RHEV_Config`